pkg/snowflake/parser.go

```diff
@@ -25,34 +25,85 @@ func NewViewSelectStatementExtractor(input string) *ViewSelectStatementExtractor
 }
 
 func (e *ViewSelectStatementExtractor) Extract() (string, error) {
-	fmt.Printf("[DEBUG] extracting view query %s\n", string(e.input))
 	e.consumeSpace()
 	e.consumeToken("create")
 	e.consumeSpace()
 	e.consumeToken("or replace")
 	e.consumeSpace()
 	e.consumeToken("secure")
 	e.consumeSpace()
+	e.consumeToken("temporary")
+	e.consumeSpace()
 	e.consumeToken("recursive")
 	e.consumeSpace()
 	e.consumeToken("view")
 	e.consumeSpace()
 	e.consumeToken("if not exists")
 	e.consumeSpace()
 	e.consumeID()
-	// TODO column list
 	e.consumeSpace()
```

Comment:
What about TODO with the column list? It doesn't apply now?

Reply:
Removed by accident; reverted.